projects/m4/011-distinct-names/js/index.js

```diff
@@ -0,0 +1,60 @@
+//011
+
+//package
+const fs = require('fs');
+const process = require('process');
+
+//variable
+const folderFiles = '\\name-files-2000-200'
+
+//function read files and elaborate them
+const readMultipleFiles = () => {
+    fs.promises.readdir(process.cwd()+folderFiles)
+        // If promise resolved
+        .then(filenames => {
+
+            let dataMtot = []
+            let dataFtot = []
+
+            let objM = []
+            let objF = []
```

Comment:
Why do you memorize the memory addresses and then modify the contents at rows 36 and 37?
.... it's your usual problem ... ;-)

Reply:
fixed!